Simplenote/Classes/UIViewController+Simplenote.swift

```diff
@@ -87,4 +87,24 @@ extension UIViewController {
             taskBlock()
         }
     }
+
+    /// Sets up a new note bar with given selector
+    ///
+    @objc
+    func configureNewNoteBar(with selector: Selector) {
```

Comment:
This API is not generic enough to grant a spot in a `UIViewController` extension. There are aroooound ~18 VCs and it's used in only two spots.

How about if this was moved over as a `UIBarButtonItem` extension instead?